### PR TITLE
Case tabs

### PR DESCRIPTION
**Pull request by taustad**

### CODE REVIEW

File: frontend/src/Views/Case/CaseProductionProfilesTabTable.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        })
+        return tableRows
+    }
+
+    useEffect(() => {
+        setRowData(profilesToRowData())
+    }, [timeSerieses])
+
+    const handleCellValueChange = (p: any) => {
+        console.log(p)
```
Comment (DanielBohme) [inline frontend/src/Views/Case/CaseProductionProfilesTabTable.tsx:70]: remove console.log

File: frontend/src/Views/Case/CaseProductionProfilesTabTable.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        properties.forEach((prop) => {
+            if (isInteger(prop)
+                && p.data[prop] !== ""
+                && p.data[prop] !== null
+                && !Number.isNaN(Number(p.data[prop]))) {
+                tableTimeSeriesValues.push({ year: parseInt(prop, 10), value: Number(p.data[prop]) })
+            }
+        })
+        tableTimeSeriesValues.sort((a, b) => a.year - b.year)
+        console.log(tableTimeSeriesValues)
```
Comment (DanielBohme) [inline frontend/src/Views/Case/CaseProductionProfilesTabTable.tsx:82]: remove console.log

File: frontend/src/Components/Case/CaseNumberInput.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { Input, Label } from "@equinor/eds-core-react"
+import { ChangeEventHandler, Dispatch, SetStateAction } from "react"
```
Comment (DanielBohme) [inline frontend/src/Components/Case/CaseNumberInput.tsx:2]: remove unused imports

File: frontend/src/Views/Case/CaseFacilitiesTab.tsx [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+    surf: Surf | undefined,
+    setSurf: Dispatch<SetStateAction<Surf | undefined>>,
+    substructure: Substructure | undefined,
+    setSubstrucutre: Dispatch<SetStateAction<Substructure | undefined>>,
+    transport: Transport | undefined,
+    setTransport: Dispatch<SetStateAction<Transport | undefined>>,
+}
+
+function CaseFacilitiesTab({
+    project, setProject,
```
Comment (DanielBohme) [inline frontend/src/Views/Case/CaseFacilitiesTab.tsx:65]: remove unused props?

File: frontend/src/Views/Case/CaseDescriptionTab.tsx [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+interface Props {
+    project: Project,
+    setProject: Dispatch<SetStateAction<Project | undefined>>,
+    caseItem: Case,
+    setCase: Dispatch<SetStateAction<Case | undefined>>,
+}
+
+function CaseDescriptionTab({
+    project,
+    setProject,
```
[the remark this answers is not in the document]
Comment (taustad) [reply frontend/src/Views/Case/CaseDescriptionTab.tsx:54]: This is a partial PR. Might need it later

File: frontend/src/Views/Case/CaseFacilitiesTab.tsx [hunk 6]
```diff
@@ -0,0 +1,336 @@
...
+        const newTransport: Transport = { ...transport }
+        newTransport.gasExportPipelineLength = Number(e.currentTarget.value)
+        setTransport(newTransport)
+    }
+
+    const handleSave = async () => {
+        if (topside) {
+            const result = await (await GetTopsideService()).newUpdate(topside)
+            setTopside(result)
+            console.log(result)
```
Comment (DanielBohme) [inline frontend/src/Views/Case/CaseFacilitiesTab.tsx:156]: remove console.log

File: frontend/src/Views/Case/CaseFacilitiesTab.tsx [hunk 7]
```diff
@@ -0,0 +1,336 @@
...
+        }
+        if (surf) {
+            const result = await (await GetSurfService()).newUpdate(surf)
+            setSurf(result)
+            console.log(result)
+        }
+        if (substructure) {
+            const result = await (await GetSubstructureService()).newUpdate(substructure)
+            setSubstrucutre(result)
+            console.log(result)
```
Comment (DanielBohme) [inline frontend/src/Views/Case/CaseFacilitiesTab.tsx:166]: remove console.log

File: frontend/src/Views/Case/CaseFacilitiesTab.tsx [hunk 8]
```diff
@@ -0,0 +1,336 @@
...
+        }
+        if (transport) {
+            const result = await (await GetTransportService()).newUpdate(transport)
+            setTransport(result)
+            console.log(result)
```
Comment (DanielBohme) [inline frontend/src/Views/Case/CaseFacilitiesTab.tsx:171]: remove console.log

**Pull request merged**